deploy/kubernetes/overlays/examples/cadvisor-args.yaml

```diff
@@ -15,5 +18,5 @@ spec:
           - --max_housekeeping_interval=15s
           - --event_storage_event_limit=default=0
           - --event_storage_age_limit=default=0
-          - --disable_metrics=percpu,disk,network,tcp,udp # enable only diskIO, cpu, memory
+          - --disable_metrics=percpu,tcp,udp              # enable only diskIO, cpu, memory, network, disk
```

Comment:
Can you add `sched` and `process` to this as well?  For reference, [here are the metrics we support](https://github.com/google/cadvisor/blob/master/container/factory.go#L46), and [here are the metrics we enable in the kubelet](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/cadvisor_linux.go#L88).  Note that we disable disk metrics when using CRI runtimes, since container disk metrics are provided through the CRI and disk metrics are the most expensive to collect.

Reply:
Done